### ISSUE TITLE
/help implementation uses i18n, which breaks on dedicated servers

### ISSUE THREAD

**Issue by Gaelan** [squeek502/AppleCore#66]
```
java.lang.NoClassDefFoundError: net/minecraft/client/resources/I18n
at squeek.applecore.commands.CommandHunger.func_71518_a(CommandHunger.java:26) ~[CommandHunger.class:?]
at com.matthewprenger.helpfixer.HelpFixer$1.func_184900_a(HelpFixer.java:47) ~[HelpFixer$1.class:1.3.14]
at net.minecraft.command.CommandHelp.func_184881_a(SourceFile:73) ~[al.class:?]
at net.minecraft.command.CommandHandler.func_175786_a(CommandHandler.java:103) [k.class:?]
at net.minecraft.command.CommandHandler.func_71556_a(CommandHandler.java:82) [k.class:?]
at net.minecraft.server.dedicated.DedicatedServer.func_71333_ah(DedicatedServer.java:421) [lb.class:?]
at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:386) [lb.class:?]
at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:611) [MinecraftServer.class:?]
at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:469) [MinecraftServer.class:?]
at java.lang.Thread.run(Thread.java:745) [?:1.8.0_91]
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.resources.I18n
at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191) ~[launchwrapper-1.12.jar:?]
at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_91]
at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_91]
... 10 more
Caused by: java.lang.NullPointerException
```

**Comment by squeek502** [comment]
Thanks for the report, should have it fixed soon.

**Comment by squeek502** [comment]
Fixed in 1.3.4
